Musee du Louvre.js

```diff
@@ -201,21 +160,36 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "http://www.louvre.fr/en/oeuvre-notices/mona-lisa-portrait-lisa-gherardini-wife-francesco-del-giocondo?sous_dept=1",
+		"url": "https://www.louvre.fr/en/oeuvre-notices/mona-lisa-portrait-lisa-gherardini-wife-francesco-del-giocondo?sous_dept=1",
 		"items": [
 			{
 				"itemType": "artwork",
 				"title": "Mona Lisa – Portrait of Lisa Gherardini, wife of Francesco del Giocondo",
-				"creators": [],
-				"extra": "Work Mona Lisa – Portrait of Lisa Gherardini, wife of Francesco del Giocondo",
-				"libraryCatalog": "Musée du Louvre",
-				"url": "http://www.louvre.fr/en/oeuvre-notices/mona-lisa-portrait-lisa-gherardini-wife-francesco-del-giocondo?sous_dept=1",
+				"creators": [
+					{
+						"lastName": "Leonardo di ser Piero da Vinci, known as LEONARDO DA VINCI (Vinci, 1452 - Amboise, 1519)",
```

Comment:
This is never going to be perfect, but could we at least remove the date of birth&death in parentheses?